### PR TITLE
Add frost

### PR DESCRIPTION
**Pull request by terezka**
For the tabs in the [class management page](https://app.zeplin.io/project/594e7e15739fd41544fe4784/screen/594e7e3a0bd405294e197ed2).

### CODE REVIEW

File: src/Nri/Colors.elm [hunk 1]
```diff
@@ -166,6 +167,16 @@ coralLightest =
 
 {-|
 
+<p style="font-size:2em; color: #eef9ff; background-color: black;">#eef9ff</p>
```
Comment (eeue56) [inline src/Nri/Colors.elm:170]: :thought_balloon: This might be a good place for elm-verify-example to make sure the colors match the actual value

Comment (eeue56) [reply src/Nri/Colors.elm:170]: @stoeffel this ties in to this issue kinda: https://github.com/stoeffel/elm-verify-examples/issues/25

Comment (stoeffel) [reply src/Nri/Colors.elm:170]: @eeue56 yes, I think so too.

**Comment by stoeffel** [comment]
I'm not sure what the plan is for migrating to the new colors. Maybe @bendansby knows if we should add new colors already and switch to the new names or if we should use the old names and only switch the color.

**Comment by bendansby** [comment]
My feeling was that we would rename and redefine pretty much in one or two fell swoops, on both the Elm and Sass sides.  Adding new colors and shifting the colors for mastery could be done as they come up or at the same time, don't feel strongly.

color transposition

**Comment by tesk9** [comment]
@bendansby are you still planning to own adding all the colors in one fell swoop?

**Comment by bendansby** [comment]
@tesk9 What would you all prefer? Any time I do something in the code base it's simply because I figure y'all have bigger fish to fry than whatever bugbear I'm tweaking. But if changing the colors fits into the current eng workflow, I'm happy to defer.

**Comment by avh4** [comment]
Is this replaced by https://github.com/NoRedInk/nri-elm-css/pull/29 ?

**Pull request closed without merging**